gst/nnstreamer/tensor_common.c

```diff
@@ -1099,6 +1099,10 @@ gst_gen_tensors_from_collectpad (GstCollectPads * collect,
     GstBuffer *buf;
 
     walk = g_slist_nth (walk, sync.data_basepad.sink_id);
+    if (walk == NULL) {
+      GST_ERROR_OBJECT (collect, "Cannot get GstCollectData from GSList");
+    }
```

Comment:
return false needed (no assertion case)

Reply:
If it is null, then walk->data should rise the error and pad as well.. Do you think it is ok to be null?

Reply:
right. Thanks.